```
Ubuntu clang-format version 13.0.1-++20220120110924+75e33f71c2da-1~exp1~20220120231001.58
test/unit/test_vector.c:849:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1337, "insert => content");
                             ^
test/unit/test_vector.c:849:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1337, "insert => content");
                              ^
test/unit/test_vector.c:850:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 1, "insert => old content");
                             ^
test/unit/test_vector.c:850:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 1, "insert => old content");
                              ^
test/unit/test_vector.c:851:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 2, "insert => old content");
                             ^
test/unit/test_vector.c:851:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 2, "insert => old content");
                              ^
test/unit/test_vector.c:852:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 3, "insert => old content");
                             ^
test/unit/test_vector.c:852:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 3, "insert => old content");
                              ^
test/unit/test_vector.c:862:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1337, "insert => content");
                             ^
test/unit/test_vector.c:862:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1337, "insert => content");
                              ^
test/unit/test_vector.c:863:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 1, "insert => old content");
                             ^
test/unit/test_vector.c:863:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 1, "insert => old content");
                              ^
test/unit/test_vector.c:864:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 2, "insert => old content");
                             ^
test/unit/test_vector.c:864:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 2, "insert => old content");
                              ^
test/unit/test_vector.c:865:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 3, "insert => old content");
                             ^
test/unit/test_vector.c:865:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 3, "insert => old content");
                              ^
test/unit/test_vector.c:874:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "insert (end) => old content");
                             ^
test/unit/test_vector.c:874:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "insert (end) => old content");
                              ^
test/unit/test_vector.c:875:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 2, "insert (end) => old content");
                             ^
test/unit/test_vector.c:875:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 2, "insert (end) => old content");
                              ^
test/unit/test_vector.c:876:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 3, "insert (end) => old content");
                             ^
test/unit/test_vector.c:876:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 3, "insert (end) => old content");
                              ^
test/unit/test_vector.c:877:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 1337, "insert (end) => content");
                             ^
test/unit/test_vector.c:877:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 1337, "insert (end) => content");
                              ^
test/unit/test_vector.c:887:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "rz_vector_insert (end, resize) => old content");
                             ^
test/unit/test_vector.c:887:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "rz_vector_insert (end, resize) => old content");
                              ^
test/unit/test_vector.c:888:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 2, "rz_vector_insert (end, resize) => old content");
                             ^
test/unit/test_vector.c:888:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 2, "rz_vector_insert (end, resize) => old content");
                              ^
test/unit/test_vector.c:889:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 3, "rz_vector_insert (end, resize) => old content");
                             ^
test/unit/test_vector.c:889:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 3, "rz_vector_insert (end, resize) => old content");
                              ^
test/unit/test_vector.c:890:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 1337, "rz_vector_insert (end, resize) => content");
                             ^
test/unit/test_vector.c:890:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 1337, "rz_vector_insert (end, resize) => content");
                              ^
test/unit/test_vector.c:904:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0xC0, "insert_range (empty) => new content");
                             ^
test/unit/test_vector.c:904:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0xC0, "insert_range (empty) => new content");
                              ^
test/unit/test_vector.c:905:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 0xFF, "insert_range (empty) => new content");
                             ^
test/unit/test_vector.c:905:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 0xFF, "insert_range (empty) => new content");
                              ^
test/unit/test_vector.c:906:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 0xEE, "insert_range (empty) => new content");
                             ^
test/unit/test_vector.c:906:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 0xEE, "insert_range (empty) => new content");
                              ^
test/unit/test_vector.c:913:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0, "insert_range => old content");
                             ^
test/unit/test_vector.c:913:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0, "insert_range => old content");
                              ^
test/unit/test_vector.c:914:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "insert_range => old content");
                             ^
test/unit/test_vector.c:914:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "insert_range => old content");
                              ^
test/unit/test_vector.c:915:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 0xC0, "insert_range => new content");
                             ^
test/unit/test_vector.c:915:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 0xC0, "insert_range => new content");
                              ^
test/unit/test_vector.c:916:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 0xFF, "insert_range => new content");
                             ^
test/unit/test_vector.c:916:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 0xFF, "insert_range => new content");
                              ^
test/unit/test_vector.c:917:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 0xEE, "insert_range => new content");
                             ^
test/unit/test_vector.c:917:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 0xEE, "insert_range => new content");
                              ^
test/unit/test_vector.c:918:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 5)), 2, "insert_range => old content");
                             ^
test/unit/test_vector.c:918:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 5)), 2, "insert_range => old content");
                              ^
test/unit/test_vector.c:925:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0, "insert_range (end) => old content");
                             ^
test/unit/test_vector.c:925:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0, "insert_range (end) => old content");
                              ^
test/unit/test_vector.c:926:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "insert_range (end) => old content");
                             ^
test/unit/test_vector.c:926:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "insert_range (end) => old content");
                              ^
test/unit/test_vector.c:927:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 2, "insert_range (end) => old content");
                             ^
test/unit/test_vector.c:927:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 2, "insert_range (end) => old content");
                              ^
test/unit/test_vector.c:928:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 0xC0, "insert_range (end) => new content");
                             ^
test/unit/test_vector.c:928:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 0xC0, "insert_range (end) => new content");
                              ^
test/unit/test_vector.c:929:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 0xFF, "insert_range (end) => new content");
                             ^
test/unit/test_vector.c:929:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 0xFF, "insert_range (end) => new content");
                              ^
test/unit/test_vector.c:930:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 5)), 0xEE, "insert_range (end) => new content");
                             ^
test/unit/test_vector.c:930:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 5)), 0xEE, "insert_range (end) => new content");
                              ^
test/unit/test_vector.c:937:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0, "insert_range (resize) => old content");
                             ^
test/unit/test_vector.c:937:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0, "insert_range (resize) => old content");
                              ^
test/unit/test_vector.c:938:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "insert_range (resize) => old content");
                             ^
test/unit/test_vector.c:938:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "insert_range (resize) => old content");
                              ^
test/unit/test_vector.c:939:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 0xC0, "insert_range (resize) => new content");
                             ^
test/unit/test_vector.c:939:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 2)), 0xC0, "insert_range (resize) => new content");
                              ^
test/unit/test_vector.c:940:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 0xFF, "insert_range (resize) => new content");
                             ^
test/unit/test_vector.c:940:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 3)), 0xFF, "insert_range (resize) => new content");
                              ^
test/unit/test_vector.c:941:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 0xEE, "insert_range (resize) => new content");
                             ^
test/unit/test_vector.c:941:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 4)), 0xEE, "insert_range (resize) => new content");
                              ^
test/unit/test_vector.c:942:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 5)), 2, "insert_range (resize) => old content");
                             ^
test/unit/test_vector.c:942:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 5)), 2, "insert_range (resize) => old content");
                              ^
test/unit/test_vector.c:955:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0, "pop => remaining content");
                             ^
test/unit/test_vector.c:955:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0, "pop => remaining content");
                              ^
test/unit/test_vector.c:956:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "pop => remaining content");
                             ^
test/unit/test_vector.c:956:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 1, "pop => remaining content");
                              ^
test/unit/test_vector.c:961:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0, "pop => remaining content");
                             ^
test/unit/test_vector.c:961:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 0, "pop => remaining content");
                              ^
test/unit/test_vector.c:979:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 1, "pop_front => remaining content");
                             ^
test/unit/test_vector.c:979:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 1, "pop_front => remaining content");
                              ^
test/unit/test_vector.c:980:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 2, "pop_front => remaining content");
                             ^
test/unit/test_vector.c:980:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 1)), 2, "pop_front => remaining content");
                              ^
test/unit/test_vector.c:985:23: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 2, "pop_front => remaining content");
                             ^
test/unit/test_vector.c:985:24: error: code should be clang-formatted [-Wclang-format-violations]
        mu_assert_eq((size_t)*((void **)rz_vector_index_ptr(&v.v, 0)), 2, "pop_front => remaining content");
                              ^
```

--- a/test/unit/test_vector.c
+++ b/test/unit/test_vector.c
@@ -837,6 +837,7 @@ static bool test_pvector_remove_at(void) {
 	mu_end;
 }
 
+// clang-format off
 static bool test_pvector_insert(void) {
 	RzPVector v;
 
@@ -992,6 +993,7 @@ static bool test_pvector_pop_front(void) {
 
 	mu_end;
 }
+// clang-format on
 
 static bool test_pvector_push(void) {
 	RzPVector v;

